README.md

```diff
@@ -148,7 +148,7 @@ releases! 🌟
    >
    > ```bash
    > # In this case, we set it to 262144:
-   > $ sudo sysctl -w vm.max_map_count=262144
+   > $ sudo systemctl -w vm.max_map_count=262144
```

Comment:
Appreciate your contribution.
Please star the project to stay tuned.

Where does this  `systemctl` come from? Do you have reference?
Do you mean [this](https://unix.stackexchange.com/questions/709586/diffence-between-systemd-sysctl-and-sysctl)

Reply:
I thought it was typo.  does sys it short for system ?

Reply:
`sysctl` is a command.